roles/lib_openshift/src/generate.py

```diff
@@ -27,19 +27,47 @@ def parse_args():
     return parser.parse_args()
 
 
+def fragment_banner(fragment_path, side, data):
+    """Generate a banner to wrap around file fragments
+
+* :param fragment_path: A path to a module fragment
+* :param side: ONE OF: "header", "footer"
+* :param data: A StringIO object to write the banner to
+"""
+    side_msg = {
+        "header": "Begin included fragment: %s",
+        "footer": "End included fragment: %s"
+    }
+    annotation = side_msg[side] % fragment_path
+
+    banner = """
```

Comment:
nit: A smaller, but highly visible banner is another option calling pit generated code blocks withoit as much real estate. Something like: ```# - - -  XXX Begin include fragment - - -```.